modules/administration-guide/partials/proc_viewing-che-metrics-on-grafana-dashboards.adoc

```diff
@@ -1,27 +1,35 @@
 // monitoring-{prod-id-short}
 
 [id="viewing-{prod-id-short}-metrics-on-grafana-dashboards"]
-= Viewing {prod-short} Server metrics on Grafana dashboards
+= Viewing {prod-short} Server from an OpenShift web console dashboard
 
-To view the {prod-short} Server metrics on Grafana:
+After configuring the in-cluster Prometheus instance to collect {prod-short} Server JVM metrics, the metrics can be viewed from a custom dashboard within the *Administrator* perspective of the OpenShift web console.
 
 .Prerequisites
 
-* Prometheus is collecting metrics on the {prod-short} cluster. See xref:monitoring-with-prometheus-and-grafana.adoc[].
+* An active `{orch-cli}` session with administrative permissions to the destination {orch-name} cluster. See {orch-cli-link}.
 
-* Grafana 6.0 or later is running on port `3000` with a corresponding Service. See link:https://grafana.com/docs/grafana/latest/installation/kubernetes/[Installing Grafana].
+* An instance of {prod-short} running in {orch-name}.
 
+* The in-cluster Prometheus instance is collecting metrics. See xref:collecting-che-metrics-with-prometheus[].
 
 .Procedure
 
-. Add the data source for the Prometheus instance. See link:https://prometheus.io/docs/visualization/grafana/#creating-a-prometheus-data-source[Creating a Prometheus data source].
+. Create a ConfigMap in the `openshift-config-managed` {orch-namespace} containing the dashboard definition and apply the necessary label. The dashboard definition is based on Grafana 6.x dashboards, however, not all Grafana 6.x dashboard features are supported within the OpenShift web console.
++
+[source,terminal,subs="+attributes,quotes"]
+----
+$ oc create configmap grafana-dashboard-devspaces-server \
+  --from-literal=devspaces-server-dashboard.json="$(curl https://raw.githubusercontent.com/eclipse-che/che-server/main/docs/grafana/openshift-console-dashboard.json)" \
+  -n openshift-config-managed
+$ oc label configmap grafana-dashboard-devspaces-server console.openshift.io/dashboard=true -n openshift-config-managed
+----
 
-. Import the example link:https://github.com/eclipse-che/che-server/blob/7.44.x/docs/grafana/dashboard.json[dashboard]. See link:https://grafana.com/docs/grafana/latest/dashboards/export-import/#import-dashboard[Import dashboard].
+.Verification steps
 
-. View the {prod-short} JVM metrics in the Grafana console:
-+
-.{prod-short} server JVM dashboard
-image::monitoring/monitoring-che-che-server-jvm-dashboard.png[The *{prod-short} server JVM* dashboard, link="{imagesdir}/monitoring/monitoring-che-che-server-jvm-dashboard.png"]
+. In the *Administrator* view of the OpenShift web console, navigate to *Observe* -> *Dashboards* from the sidebar menu on the left.
```

Comment:
Our style guide recommends avoiding positional references in the UI like "the sidebar menu on the left", also IBM Style advises avoiding "navigate":
```suggestion
. In the *Administrator* view of the OpenShift web console, go to *Observe* -> *Dashboards*.
```